### PR TITLE
Add more unit tests for ./store/sqlstore/user.go

### PR DESCRIPTION
**Pull request by yedamao**
#### Summary

Add more unit tests for ./store/sqlstore/user.go

#### Ticket Link

Fixes https://github.com/mattermost/focalboard/issues/430

### CODE REVIEW

File: server/services/store/storetests/users.go [hunk 1]
```diff
@@ -52,3 +68,99 @@ func testGetWorkspaceUsers(t *testing.T, store store.Store) {
...
+func testCreateAndGetUser(t *testing.T, store store.Store) {
+	user := &model.User{
+		ID:       uuid.New().String(),
+		Username: "damao",
+		Email:    "mock@email.com",
+	}
+
+	t.Run("CreateUser", func(t *testing.T) {
+		err := store.CreateUser(user)
+		require.Nil(t, err)
```
Comment (wiggin77) [inline server/services/store/storetests/users.go:81]:
```suggestion
		require.NoError(t, err)
```
Same for the other instances below.

**Review by wiggin77 — CHANGES_REQUESTED** [review]
Looks great @yedamao .  One minor request... use `require.NoError` when checking error cases.

**Comment by yedamao** [comment]
@wiggin77 PTAL

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Pull request merged**